src/diffusers/pipelines/ddpm/pipeline_ddpm.py

```diff
@@ -68,9 +68,9 @@ def __call__(
         """
         # Sample gaussian noise to begin loop
         if isinstance(self.unet.sample_size, int):
-            image_shape = (batch_size, self.unet.in_channels, self.unet.sample_size, self.unet.sample_size)
+            image_shape = (batch_size, self.unet.config.in_channels, self.unet.sample_size, self.unet.sample_size)
```

Comment:
Why is `sample_size` not accessed through the `config` too?

Reply:
Good point!